docs/UserGuide.md

```diff
@@ -90,11 +90,106 @@ This marks the end of the quick start. Click
 
 ### 3.2 Application Home Window Breakdown
 
-### 3.3 Client Explanation
+Here is an overview of the different elements that makes up the Home Window of DonnaFin
 
-### 3.4 Application Client Window Breakdown
+![MainWindowScreenshot.png](./images/MainWindowScreenshot.png)
 
-### 3.5 Command Format
+#### 3.2.1 Quick Access Buttons
+* This area gives you access to the File button that allows the you to quickly exit the app, and the help 
+  button that redirects you to the User Guide for further guidance.
+
+#### 3.2.2 Command Box
+* The command box is where you can enter your commands in.
+
+#### 3.2.3 Results Display
+* The results display is where the action taken by DonnaFin in response to the command you keyed in is displayed.
```

Comment:
The result display is where the response to the command entered is displayed.? More concise